packages/components/package.json

```diff
@@ -1,35 +1,40 @@
 {
   "name": "@terra-ui-packages/components",
   "version": "0.0.2",
```

Comment:
Need to bump version.

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4116/commits/d86f15c26c593a76ddb92c6187a3d08d04c413a6.